lib/forest_liana/collections/company.rb

```diff
@@ -7,6 +7,15 @@ class Forest::Company
 
   action 'Mark as Live'
 
+  action 'New emitted transaction', fields: [{
```

Comment:
Shouldn't actions name start with a verb to be consistent with our current convention?